lint/highlight.py

```diff
@@ -389,6 +393,35 @@ def draw(self, view):
         """
         self.set_mark_style()
 
+        _stylesheet = '''
```

Comment:
@rchl i copy/paste it from the `Packages/Default/exec.py` 
ok i will update the code now